asset-transfer/test/test_AssetTransfer.js

```diff
@@ -16,9 +16,9 @@ const args = process.argv;
 process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0'
 
 
-const deployment_address = 'ADDRESS_PLACEHOLDER';
-const username = 'USER_PLACEHOLDER';
-const pass = 'PASSWORD_PLACEHOLDER';
+const deployment_address = 'http://localhost/api/concord/eth';
+const username = 'admin@blockchain.local';
+const pass = 'T3sting!';
```

Comment:
Don't think you wanted to submit this?

Reply:
No, this was a mistake. I will close this PR.